### PR TITLE
[ACTION] Google Sheets: Added new action insert-comment

### PR DESCRIPTION
**Pull request by jcortes**

### CODE REVIEW

File: components/google_sheets/actions/insert-comment/insert-comment.mjs [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import app from "../../google_sheets.app.mjs";
+
+export default {
+  key: "google_sheets-insert-comment",
+  name: "Insert Comment",
+  description: "Insert a comment into a spreadsheet. [See the docs here](https://developers.google.com/drive/api/v3/reference/comments/create)",
+  version: "0.0.3",
```
Comment (luancazarine) [inline components/google_sheets/actions/insert-comment/insert-comment.mjs:7]:
```suggestion
  version: "0.0.1",
```

**Review by luancazarine — CHANGES_REQUESTED** [review]
Hi @jcortes. Thank you for your contribution! I think you forgot to reset the version.

**Review by luancazarine — APPROVED** [review]
LGTM!

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release!

**Pull request merged**